pkg/apis/core/types.go

```diff
@@ -1941,6 +1942,16 @@ const (
 	PullIfNotPresent PullPolicy = "IfNotPresent"
 )
 
+// PreemptionPolicy describes a policy for if/when to preempt a pod.
+type PreemptionPolicy string
+
+const (
+	// PreemptLowerPriority means that pod can preempt other pods with lower priority.
+	PreemptLowerPriority PreemptionPolicy = "PreemptLowerPriority"
```

Comment:
@bsalamat, did you want this to be named `LowerPriority`?

Reply:
@bsalamat  is PreemptLowerPriority  easier to understand than LowerPriority？WDYT？

Reply:
I don't have a strong preference, but I thought since the field name is "PreemptionPolicy", we can be more succinct for the value and call it just "LowerPriority", but "PreemptLowerPriority" is fine too.